databases/week3/mongoDB/databaseQueriesMeal.mongodb

```diff
@@ -0,0 +1,165 @@
+// Select the database to use.
+use('mealSharingApp');
+
+// Run a find command to get all meal
+use('mealSharingApp');
+db.Meals.find();
+
+// Add a new meal
+use('mealSharingApp');
+db.Meals.insertOne([
+  {
+    'id': 7, 
+    'title': 'Daal Bati Churma',
+    'description': 'Delicious Indian dish made in ghee',
+    'location': 'Aarhus',
+    'When': new Date ('2023-04-10T11:30:00Z'),
+    'max_reservation' : 30 , 
+    'price' : 250,
+    'created_date' : new Date ('2023-03-08')
+  }
+])
+
+use('mealSharingApp');
+db.Meals.find();
+
+//Get a meal with any id, fx 1
+use('mealSharingApp');
+db.Meals.find({'id': 3});
+
+
+
+//Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+use('mealSharingApp');
+db.Meals.updateOne( { 'title': "Butter Chicken Curry" }, { $set: { 'title': "Buttery Butter Chicken Curry" } } ) 
+
+use('mealSharingApp');
+db.Meals.find();
+
+
+//Delete a meal with any id, fx 1
+use('mealSharingApp');
+db.Meals.deleteOne({ 'title': "Buttery Butter Chicken Curry" })
+
+use('mealSharingApp');
+db.Meals.find();
+
+//Get meals that has a price smaller than a specific price fx 90
+use('mealSharingApp');
+db.Meals.find({ 'price': {$lt : 250} });
+
+
+//Get meals that still has available reservations
+
+
+//Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+use('mealSharingApp');
+db.Meals.find({ title: /Daal/ });
```

Comment:
If you want to perform a case-insensitive search, you can add the `i` flag to the regular expression:
db.Meals.find({ title: /Daal/i });